website/src/documentation/transforms/java/element-wise/filter.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: section
+title: "Filter"
+permalink: /documentation/transforms/java/elementwise/filter/
+section_menu: section-menu/documentation.html
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Filter
+<table align="left">
+    <a target="_blank" class="button"
+        href="https://beam.apache.org/releases/javadoc/current/index.html?org/apache/beam/sdk/transforms/Filter.html">
+      <img src="https://beam.apache.org/images/logos/sdks/java.png" width="20px" height="20px"
+           alt="Javadoc" />
+     Javadoc
+    </a>
+</table>
+<br>
+Given a predicate, filter out all elements that don't satisfy that predicate.
+May also be used to filter based on an inequality with a given value based
+on the comparison ordering of the element.
+
+## Examples
+**Example 1**: Filtering with a predicate
+
+```java
+PCollection<String> allStrings = Create.of("Hello", "world", "hi");
+PCollection<String> longStrings = allStrings
+    .apply(Filter.by(new SerializableFunction<String, Boolean>() {
+      @Override
+      public Boolean apply(String input) {
+        return input.length() > 3;
+      }
+    }));
+```
```

Comment:
I agree. But @davidcavazos is replacing these samples with his colab in the new few months, so it might make more sense to invest time into that.

These snippets were written by @kennknowles and @bjchambers a while ago.